sql/src/main/java/io/crate/executor/transport/TransportActionProvider.java

```diff
@@ -113,6 +121,8 @@ public TransportActionProvider(Provider<TransportFetchNodeAction> transportFetch
         this.transportCreateSnapshotActionProvider = transportCreateSnapshotActionProvider;
         this.transportRestoreSnapshotActionProvider = transportRestoreSnapshotActionProvider;
         this.transportGetSnapshotsActionProvider = transportGetSnapshotsActionPovider;
+        this.transportOpenIndexActionProvider = transportOpenIndexActionProvider;
+        this.transportCloseIndexActionProvider = transportCloseIndexActionProvider;
```

Comment:
Could we inject these dependencies directly into the AlterTableOperation - avoiding the TransportActionProvider roundtrip?